site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
+
+## [Volume Slider](https://github.com/citra-emu/citra/pull/3831) by [Fearlesstobi](https://github.com/fearlesstobi)
+
+You can now configure the volume level of Citra directly from the configuration
+window. This allows you to set an individual volume level for Citra without
+having to manually set it in your operating system settings.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/volume_slider.png"
+    title="Volume Slider" >}}
+
+## [Discord Rich Presence](https://github.com/citra-emu/citra/pull/3883) by [CaptV0rt3x](https://github.com/CaptV0rt3x)
+
+Rich Presence allows you to leverage the totally overhauled "Now Playing"
+section in a Discord user's profile. This feature lets users show off
+when they are playing their favorite games on Citra. This has been a common
+feature request from many users and since many other emulators have already
+implemented this, it made sense to implement it here as well.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/rpc.png"
+        title="Discord Rich Presence" >}}
+
+Discord Rich Presence works by creating a local connection with Citra and your
+Discord App. It ***does not*** connect to (or) send any data to discord servers.
+This option is user configurable, so you can choose to show or not to show rich
+presence in your discord status. Just head out to <kbd><samp>Emulation</samp> &rarr; <samp>Configure...</samp> &rarr; <samp>Web</samp> &rarr; <samp>Discord Presence</samp></kbd> and uncheck
+"Show current game in your Discord status".
+
+## [Set Default values for settings](https://github.com/citra-emu/citra/pull/3924) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+The majority of support requests we get are mostly due to users accidentally
+enabling or disabling some settings. But now with this implementation, we have
+default values for all our settings. This ensures that in case of any new
+settings changes, majority of the users (who haven't customized those settings)
+will now have the new default value automatically.
+
+## [Add support for encrypted games](https://github.com/citra-emu/citra/pull/4020) by [wwylele](https://github.com/wwylele)
+
+You can now directly use your encrypted game dumps in Citra, without having to
+decrypt them. This requires that the user provide their keys via a text file
+in `sysdata/aes_keys.txt`.
+
+Currently all known encryption methods are implemented except for the Seed
+crypto. Seed crypto is used in newer eshop games, which essentially uses an
+additional title-unique key that is stored outside of the ROM.
+
+## [Scripting support](https://github.com/citra-emu/citra/pull/4016) by [EverOddish](https://github.com/EverOddish)
+
+Scripting is an extremely useful feature which allows us to do things like
+debugging emulator internals, reverse engineering 3DS game internals,
+Tool-Assisted-Speedrunning (TAS), and cheat in games.
+(**Does not add actual Gameshark / AR Code cheat support**)
```

Comment:
This doesn't fit well as a comment. Wrap it into the paragraph itself.
e.g.
`[...] and the ability to manipulate memory in games. A cheat-like interface could be implemented through this (though, not specifically Gameshark / AR codes).`